tutoraspects/templates/openedx-assets/assets/charts/Transcripts_Captions_Per_Video.yaml

```diff
@@ -0,0 +1,70 @@
+      filterable: true
+      groupby: true
+      id: 53
+      is_certified: false
+      is_dttm: false
+      python_date_format: null
+      type: String
+      type_generic: 1
+      verbose_name: null
+      warning_markdown: null
+    expressionType: SIMPLE
+    hasCustomLabel: true
+    isNew: false
+    label: Unique transcript users
+    optionName: metric_w1zxo19fl_4jlrzti9ec6
+    sqlExpression: null
+  order_desc: true
+  rich_tooltip: true
+  row_limit: 10000
+  show_legend: true
+  slice_id: 47
+  time_range: 'DATEADD(DATETIME("now"), -1, month) : DATEADD(DATETIME("now"), 1, day)'
+  viz_type: dist_bar
+  x_axis_label: Video Title
+  x_ticks_layout: auto
+  y_axis_bounds:
+  - null
+  - null
+  y_axis_format: SMART_NUMBER
+  y_axis_label: Number of Users / Uses
+query_context: '{"datasource":{"id":3,"type":"table"},"force":false,"queries":[{"time_range":"DATEADD(DATETIME(\"now\"),
+  -1, month) : DATEADD(DATETIME(\"now\"), 1, day)","granularity":"emission_time","filters":[],"extras":{"having":"","where":""},"applied_time_extras":{},"columns":["video_name"],"metrics":["count",{"aggregate":"COUNT_DISTINCT","column":{"advanced_data_type":null,"certification_details":null,"certified_by":null,"column_name":"actor_id","description":null,"expression":null,"filterable":true,"groupby":true,"id":53,"is_certified":false,"is_dttm":false,"python_date_format":null,"type":"String","type_generic":1,"verbose_name":null,"warning_markdown":null},"expressionType":"SIMPLE","hasCustomLabel":true,"isNew":false,"label":"Unique
+  transcript users","optionName":"metric_w1zxo19fl_4jlrzti9ec6","sqlExpression":null}],"annotation_layers":[],"row_limit":10000,"series_limit":0,"order_desc":true,"url_params":{},"custom_params":{},"custom_form_data":{}}],"form_data":{"datasource":"3__table","viz_type":"dist_bar","slice_id":47,"granularity_sqla":"emission_time","time_range":"DATEADD(DATETIME(\"now\"),
+  -1, month) : DATEADD(DATETIME(\"now\"), 1, day)","metrics":["count",{"aggregate":"COUNT_DISTINCT","column":{"advanced_data_type":null,"certification_details":null,"certified_by":null,"column_name":"actor_id","description":null,"expression":null,"filterable":true,"groupby":true,"id":53,"is_certified":false,"is_dttm":false,"python_date_format":null,"type":"String","type_generic":1,"verbose_name":null,"warning_markdown":null},"expressionType":"SIMPLE","hasCustomLabel":true,"isNew":false,"label":"Unique
+  transcript users","optionName":"metric_w1zxo19fl_4jlrzti9ec6","sqlExpression":null}],"adhoc_filters":[],"groupby":["video_name"],"columns":[],"row_limit":10000,"order_desc":true,"color_scheme":"supersetColors","show_legend":true,"rich_tooltip":true,"y_axis_format":"SMART_NUMBER","y_axis_label":"Number
+  of Users / Uses","y_axis_bounds":[null,null],"x_axis_label":"Video Title","bottom_margin":"auto","x_ticks_layout":"auto","extra_form_data":{},"dashboards":[23],"force":false,"result_format":"json","result_type":"full"},"result_format":"json","result_type":"full"}'
```

Comment:
What's this `query_context` about?

Is it about the upgrade of Superset?

Do we need it?

Reply:
I wasn't sure, but it seemed to start showing up with the 2.1 upgrade. @SoryRawyer have you run across these? I think I will delete it to see if it breaks anything because it does have some hard coded English strings in there that we definitely don't want showing up.

Reply:
Ah yeah I've seen this since the upgrade but I've been able to delete it without noticing any issues.